.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+/dist
```

Comment:
I'm curious - what within this project generates a `/dist` directory? The default build goes to `/package` doesn't it?

Reply:
this was the default for tsup so i added it when i was  messing around.